src/DynamicRupture/Output/Builders/GeoGeometryBuilder.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef SEISSOL_GEO_GEOMETRY_BUILDER_HPP
+#define SEISSOL_GEO_GEOMETRY_BUILDER_HPP
+
+#include "Geometry/MeshReader.h"
+
+namespace seissol::dr::output {
+class GeometryBuilder {
```

Comment:
Should inherit from `OutputBuilder`, shouldn't it?

Reply:
Renamed the class to `IntegratedOutputBuilder`.

I don't know. In general they only shared `setMeshReader` method. In theory we can create a common abstract class (interface) with only `setMeshReader` method. But I don't think that it makes too much sense.

What do you think?